mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/config/MasterConfiguration.java

```diff
@@ -386,4 +386,8 @@ default Duration getHeartbeatInterval() {
     default Duration getMaxAssignmentThreshold() {
         return Duration.ofMillis(getAssignmentIntervalInMs());
     }
+
+    @Config("mantis.route.namedjobs.useLaunched")
+    @Default("false")
+    boolean getNamedJobsReferToLaunched();
```

Comment:
Do we not need this to be a fast property?

Reply:
fast property would be nice but I think having it as a config is also okay.